Action.c

```diff
@@ -530,8 +530,9 @@ static Htop_Reaction actionHelp(State* st) {
    attrset(CRT_colors[DEFAULT_COLOR]);
    mvaddstr(line++, 0, "Swap bar:      ");
    addattrstr(CRT_colors[BAR_BORDER], "[");
-   addattrstr(CRT_colors[SWAP], "used");
-   addattrstr(CRT_colors[BAR_SHADOW], "                                          used/total");
+   addattrstr(CRT_colors[SWAP], "used"); addstr("/");
```

Comment:
Shouldn't the `/` be marked in `BAR_SHADOW` for consistency?

Reply:
This fulfils consistency (I am not saying the old behaviour should persist) :
https://github.com/htop-dev/htop/blob/bfcb8ca0196eef942e6363e2fd7faa80eddec644/Action.c#L505-L519

Reply:
Ah, k.

Reply:
I agree that the `/` color ought to be `BAR_SHADOW` but I didn't want to then fix the non-swap meter help output in what should just be a "swap cache" pull request. Can I do a followup PR instead that focuses on fixing all of the help output?